setup.py

```diff
@@ -12,7 +12,7 @@
     from setuptools import setup
 
 install_requires = [
-    'Django>=1.8,<1.12',
+    'Django>=1.11,<2.1',
```

Comment:
I wouldn't specify an upper bound -- it might be that this will work fine with Django 2.1 without any changes.